thehaguetech/thehaguetech-site
# Feature: Meta info for dynamic pages

This PR brings meta info to the Event Details & Story Details pages.

Test it like this:

- Go to https://metatags.io/
- Fill in URL: https://www.thehaguetech.com/stories/announcing-the-digital-competence-and-education-center
- Check if meta data is right

It should show something like this:

![image](https://user-images.githubusercontent.com/899234/80155471-11fa7e00-85c2-11ea-962d-376ec019d8b9.png)
